src/app/containers/IndexPageSection/utilities/storyRowsSplitter.jsx

```diff
@@ -23,11 +23,17 @@ const splitNoImageRow = standardSlices => [
   standardSlices.slice(2),
 ];
 
-const getRows = (items, isFirstSection) => {
+const getRows = ({ items, isFirstSection, showAllPromos }) => {
   const [firstRow, unsplitregularItems] = splitFirstRow(items, isFirstSection);
-  const [regularRows, [noImageRow]] = splitNoImageRow(
-    splitStandardRows(unsplitregularItems),
-  );
+  let regularRows = [];
+  let noImageRow = [];
+  if (showAllPromos) {
+    regularRows = splitStandardRows(unsplitregularItems);
+  } else {
+    [regularRows, [noImageRow]] = splitNoImageRow(
+      splitStandardRows(unsplitregularItems),
+    );
+  }
```

Comment:
I'd like to refactor this a little bit to make it a bit easier to read / maintain.

```suggestion
  let regularRows = [];
  let noImageRow = [];
  const standardRows = splitStandardRows(unsplitregularItems);
  
  if (showAllPromos) {
    regularRows = standardRows;
  } else {
    [regularRows, [noImageRow]] = splitNoImageRow(standardRows);
  }
```